### DISCUSSION TITLE
引用后面的空格怎么去掉？\cite

### DISCUSSION THREAD

**Discussion by xhuan8** [tuna/thuthesis#944] [Q&A]
\usepackage[nospace,noadjust]{cite}

原本前后都有空格的，现在通过这种方式把前的空格去掉了，但是后面还是有空格。
中文论文怎么去掉后面的空格？

等人[12] 使用

**Comment by Harry-Chen** [comment]
在中西文混排时，在不同文种之间通常会插入一些间距以改善视觉效果。这不是语义上的“空格”，不应当移除。

**Comment by zepinglee** [comment]
`cite` 跟 `natbib` 冲突的吧？你是怎么编译通过的？

**Reply by xhuan8** [reply]
我用的版本比较老，用 \textsuperscript变成上标的

**Comment by xhuan8** [comment]
在cite后面加了 \hspace{1sp}解决了
